Polyfills/Canvas/Source/Image.cpp

```diff
@@ -85,32 +86,32 @@ namespace Babylon::Polyfills::Internal
         UrlLib::UrlRequest request{};
         request.Open(UrlLib::UrlMethod::Get, text);
         request.ResponseType(UrlLib::UrlResponseType::Buffer);
-        request.SendAsync().then(m_runtimeScheduler, arcana::cancellation::none(), [env{ info.Env() }, this, request{std::move(request)}](arcana::expected<void, std::exception_ptr> result) {
+        request.SendAsync().then(m_runtimeScheduler, *m_cancellationSource, [env{info.Env()}, this, request{std::move(request)}](arcana::expected<void, std::exception_ptr> result) {
             if (result.has_error())
             {
-                throw Napi::Error::New(env, result.error());
+                Napi::Error::New(env, result.error()).ThrowAsJavaScriptException();
```

Comment:
This is the same. It will still be an uncaught exception. Does canvas image not have a mechanism for errors?

Reply:
I've found nothing in the doc or in the GUI image code.